src/Message/RefundRequest.php

```diff
@@ -50,6 +50,16 @@
  */
 class RefundRequest extends AbstractRequest
 {
+    public function getRefundApplicationFee()
+    {
+        return $this->getParameter('refundApplicationFee');
+    }
+
+    public function setRefundApplicationFee($value)
+    {
+        return $this->setParameter('refundApplicationFee', $value);
+    }
+
```

Comment:
I thought it might cause confusion so I did put it in README but I've added docblocks for it now too.